src/client/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -461,7 +461,9 @@ public static IAuthorityValidator CreateAuthorityValidator(AuthorityInfo authori
             ///
             /// - if the authority is not defined at the application level and the request level is not AAD, use the request authority
             /// - if the authority is defined at app level, and the request level authority of is of different type, throw an exception
-            ///
+            /// 
+            /// - if the intended authority is consumers, please define it at the app level and not at the request level. 
```

Comment:
has this been tested for the scenario mentioned in the bug?

Reply:
Good point. Tests pushed.